apple/internal/rule_factory.bzl

```diff
@@ -137,6 +137,11 @@ _COMMON_PRIVATE_TOOL_ATTRS = dicts.add(
             allow_single_file = True,
             default = Label("@build_bazel_rules_apple//apple/internal/templates:ios_sim_template"),
         ),
+        "_macos_runner_template": attr.label(
+            cfg = "host",
+            allow_single_file = True,
+            default = Label("@build_bazel_rules_apple//apple/internal/templates:macos_sim_template"),
+        ),
```

Comment:
The above `_runner_template` mentions supporting multiple platforms. I'm not sure what the exact solution they were imagining was so I just added another entry here.